[2021-05-08 18:45:14,724] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: dag_4.move_files 2021-05-07T00:00:00+00:00 [queued]>
[2021-05-08 18:45:14,742] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: dag_4.move_files 2021-05-07T00:00:00+00:00 [queued]>
[2021-05-08 18:45:14,743] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2021-05-08 18:45:14,743] {taskinstance.py:1069} INFO - Starting attempt 9 of 11
[2021-05-08 18:45:14,743] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2021-05-08 18:45:14,872] {taskinstance.py:1089} INFO - Executing <Task(PythonOperator): move_files> on 2021-05-07T00:00:00+00:00
[2021-05-08 18:45:14,881] {standard_task_runner.py:52} INFO - Started process 193515 to run task
[2021-05-08 18:45:14,887] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'dag_4', 'move_files', '2021-05-07T00:00:00+00:00', '--job-id', '87', '--pool', 'default_pool', '--raw', '--subdir', '/home/navneetsajwan/airflow/dags/dag__4.py', '--cfg-path', '/tmp/tmp0kxyrtjk', '--error-file', '/tmp/tmp_lzv7m5p']
[2021-05-08 18:45:14,887] {standard_task_runner.py:77} INFO - Job 87: Subtask move_files
[2021-05-08 18:45:15,052] {logging_mixin.py:104} INFO - Running <TaskInstance: dag_4.move_files 2021-05-07T00:00:00+00:00 [running]> on host navneetsajwan-Lenovo-G50-80
[2021-05-08 18:45:15,092] {taskinstance.py:1281} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=navneet@metaorigins.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dag_4
AIRFLOW_CTX_TASK_ID=move_files
AIRFLOW_CTX_EXECUTION_DATE=2021-05-07T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-05-07T00:00:00+00:00
[2021-05-08 18:45:15,094] {python.py:118} INFO - Done. Returned value was: None
[2021-05-08 18:45:15,101] {taskinstance.py:1185} INFO - Marking task as SUCCESS. dag_id=dag_4, task_id=move_files, execution_date=20210507T000000, start_date=20210508T131514, end_date=20210508T131515
[2021-05-08 18:45:15,371] {taskinstance.py:1246} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-05-08 18:45:15,379] {local_task_job.py:146} INFO - Task exited with return code 0
